src/legacy/core_plugins/kibana/public/home/np_ready/components/sample_data/index.tsx

```diff
@@ -42,7 +42,7 @@ interface Props {
 export function SampleDataCard({ urlBasePath, onDecline, onConfirm }: Props) {
   return (
     <EuiCard
-      image={`${urlBasePath}/plugins/kibana/home/assets/illo_dashboard.png`}
+      image={`${urlBasePath}/plugins/kibana/home/assets/illo_telemetry.png`}
```

Comment:
nit: Can we rename this to `sample_data_illustration.png` or something? Not really telemetry related anymore

Reply:
good catch!